tests/Makefile

```diff
@@ -141,6 +148,17 @@ lcs-check: $(LCS_ENGINE)
 	$$cmd
 
 ################################################################
+# LCB compiler tests
+################################################################
+
+export LC_COMPILE
+compiler-check: $(LCS_ENGINE)
+	@rm -f $(LCS_LOG)
+	@cmd="$(LCC_CMD)"; \
+	echo "$$cmd" $(_PRINT_RULE); \
+	$$cmd
```

Comment:
- I'm not sure that this rule needs to have `($LCS_ENGINE)` as a dependency
- I would normally incorporate setting the environment variable into the script
- I think you're deleting the LiveCode Script tests' logfile -- maybe define an `LCC_LOG` make variable

```make
compiler-check:
	@rm -f $(LCC_LOG)
	@export LC_COMPILE="$(LC_COMPILE)"; \
	cmd="$(LCC_CMD)"; \
	echo "$$cmd" $(_PRINT_RULE); \
	$$cmd
```